xarray/tests/test_interp.py

```diff
@@ -62,6 +62,43 @@ def get_example_data(case: int) -> xr.DataArray:
         raise ValueError("case must be 1-4")
 
 
+@pytest.fixture
+def nd_interp_coords():
+    # interpolation indices for nd interpolation of da from case 3 of get_example_data
+
+    da = get_example_data(case=3)
+
+    coords = {}
+    # grid -> grid
+    coords["xdestnp"] = np.linspace(0.1, 1.0, 11)
+    coords["ydestnp"] = np.linspace(0.0, 0.2, 10)
+    coords["zdestnp"] = da.get_index("z")  # type: ignore[assignment]
```

Comment:
```suggestion
    coords["zdestnp"] = da.z.data
```

Is this equivalent?

Reply:
I guess using `get_index` keeps the `pandas.index` wrapper so your way is better